spec/models/canonical/ingredients_alchemical_property_spec.rb

```diff
@@ -30,15 +30,15 @@
           create(
             :canonical_ingredients_alchemical_property,
             priority:   1,
```

Comment:
I don't like that this maintains the extra 2 spaces to theoretically align values even though the value is omitted on the next line. I've found lots of issues with this Rubocop cop in general and am thinking of changing the required style to _not_ align values in these situations, but that's a change for another PR.